tests/Integration/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminControllerTest.php

```diff
@@ -102,6 +102,7 @@ protected function setUp(): void
             ->disableOriginalConstructor()
             ->getMock();
         $shopMock->id = 1;
+        $shopMock->id_category = 1;
```

Comment:
not sure why this haven't failed in other pull requests, I don't see how its associated with my pr

Reply:
That's weird..? what was failing exactly?

Reply:
Failed due to wrong type if I remember correctly (null instead of int provided in constructor of ProductRepository. Argument is provided like this in yml 
```
prestashop.adapter.product.repository.product_repository:
...
  - '@=service("prestashop.adapter.legacy.context").getContext().shop.id_category')
```